### PR TITLE
Sanitize alias, contacts, mailboxes and users before creating them

### PR DESCRIPTION
**Pull request by acasajus**

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1802,18 +1805,22 @@ def create(cls, **kw):
         commit = kw.pop("commit", False)
         flush = kw.pop("flush", False)
 
-        new_contact = cls(**kw)
-
-        website_email = kw["website_email"]
         # make sure email is lowercase and doesn't have any whitespace
-        website_email = sanitize_email(website_email)
+        website_email = sanitize_email(kw["website_email"]).replace("\u200f", "")
```
Comment (nguyenkims) [inline app/models.py:1809]: can we add `replace("\u200f", "")` to `sanitize_email()`?

File: app/models.py [hunk 2]
```diff
@@ -2599,6 +2607,26 @@ def aliases(self) -> [Alias]:
 
         return ret
 
+    @classmethod
+    def create(cls, **kw):
+        # whether to call Session.commit
+        commit = kw.pop("commit", False)
+        flush = kw.pop("flush", False)
+
+        if "email" in kw:
```
Comment (nguyenkims) [inline app/models.py:2616]: can we call Base.create() after sanitizing the email that will take care of the commit and flush options?

Comment (acasajus) [reply app/models.py:2612]: Done

File: crontab.yml [hunk 3]
```diff
@@ -2,64 +2,66 @@ jobs:
   - name: SimpleLogin growth stats
     command: python /code/cron.py -j stats
     shell: /bin/bash
-    schedule: "0 0 * * *"
```
Comment (nguyenkims) [inline crontab.yml:5]: For this one can we keep the same hour so we can compare it to past data?

Comment (acasajus) [reply crontab.yml:5]: Sure. But 15m should not make a noticeable difference.

**Review by nguyenkims — APPROVED** [review]

**Pull request merged**